xbmc/platform/darwin/ios/Info.plist.in

```diff
@@ -251,7 +251,7 @@
 		<integer>2</integer>
 	</array>
 	<key>CFBundleIdentifier</key>
-	<string>org.xbmc.@APP_NAME_LC@-ios</string>
+	<string>@PLATFORM_BUNDLE_IDENTIFIER@</string>
```

Comment:
What's the reason to add `-${CORE_PLATFORM_NAME_LC}` at the end of `${APP_PACKAGE}` instead of just using `${APP_PACKAGE}`?

Reply:
Legacy more than anything to be honest. Thats the layout thats currently used. Defaults back to current setup if you dont provide your own -DBUNDLE_IDENTIFIER_DESC flag. The only other reason would be to differentiate between tvos/ios certs/apps, but come catalina, apple is changing their certs to be generic from what ive read. 
Happy to change to whatever you want really.

Reply:
`-DPLATFORM_BUNDLE_IDENTIFIER` actually ;)